### DISCUSSION TITLE
Weird behaviour

### DISCUSSION THREAD

**Discussion by parkercz** [trakt/showly#733] [General]
Ok i put tv shows into Add to my shows because if i add it to watch list its not in progress tab. Also i install showly on tablet and sync it and tons of shows missing that i dont watch single episode from.
Why is that not synced ? We need some kind backup for showly to put it on other device without trakt.

**Comment by michaldrabik** [comment]
This happens because Trakt does not recognize something like My Shows section where you can add show and have 0 watched episodes. Trakt progress is only registered when at least 1 episode is marked as watched.

So when you add a new show into My Shows in Showly and export to Trakt you will not see it registered anyhow on their website.

As for backup I'm aware - hopefully I'll be able to add it in the future. Regards
